app/views/hyrax/dashboard/sidebar/_activity.html.erb

```diff
@@ -0,0 +1,31 @@
+<li class="h5"><h3 class="h2 lato-p white sidebar-header"><%= t('hyrax.admin.sidebar.activity') %></h3></li>
```

Comment:
`.h2` here adds too much text spacing and causes some cut offs too. Removing `.h2` works better